<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoPage</class>
 <widget class="QWidget" name="infoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>infoPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Your data:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="firstName">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>FirstName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LastName">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LastName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>PLN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>USD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>EUR</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="SecondName">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SecondName</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Your Account Number is:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="accNumber">
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>14214142141421</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>infoPage</receiver>
   <slot>passwordChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>infoPage</receiver>
   <slot>goBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>infoPage</receiver>
   <slot>changeType()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>passwordChange()</slot>
  <slot>goBack()</slot>
  <slot>changeType()</slot>
 </slots>
</ui>
